Validation the menus y tests

Saex05 opened the pull request:
Validation 01

Saex05 committed:
Validation the menus y tests

Miquilin commented on src/com/jalasoft/search_files/utils/borrador.py:
```diff
@@ -0,0 +1,43 @@
+from src.com.jalasoft.search_files.utils.validation import Validation
```
please remove this file and add a unitest for all the classes created

Miquilin commented on src/com/jalasoft/search_files/utils/validation.py:
```diff
@@ -0,0 +1,59 @@
+import os
+import fnmatch
```
Apparently it is no being used, please remove

Miquilin commented on src/com/jalasoft/search_files/utils/validation.py:
```diff
@@ -0,0 +1,59 @@
+    def __init__(self):
+        pass
+
+    def Isnumber(self, cadena):
```
please change it to lowercase, just like:
def is_number

Miquilin commented on src/com/jalasoft/search_files/utils/validation.py:
```diff
@@ -0,0 +1,59 @@
+        responseBool = cadena.isdigit()
+        return responseBool
+
+    def isTheRange(self, number, upperLimit, lowerLimit):
```
please change it to lowercase, just like:
def is_the_range(self, number, upper_limit, lower_limit)

Miquilin commented on src/com/jalasoft/search_files/utils/validation.py:
```diff
@@ -0,0 +1,59 @@
+        else:
+            return False
+
+    def AttemptMaximum(self, intentos):
```
please change it to lowercase, just like:
def attempt_maximum..

Miquilin commented on src/com/jalasoft/search_files/utils/validation.py:
```diff
@@ -0,0 +1,59 @@
+        else:
+            return False
+
+    def isPathValid(self, directory):
```
please change it to lowercase, just like:
def is_path_valid

Miquilin commented on src/com/jalasoft/search_files/utils/validation.py:
```diff
@@ -0,0 +1,59 @@
+        directory = str(directory)
+        return os.path.exists(directory)
+
+    def hasValidCharacters(self, cadena):
```
please change it to lowercase, just like:
def has_valid_characters...

Apparently this method should be removed due to the "hasValidCharacters1" method has the same implementation and it is doing another validation more.

Miquilin commented on src/com/jalasoft/search_files/utils/validation.py:
```diff
@@ -0,0 +1,59 @@
+        cadena = str(cadena)
+        return cadena.isalnum()
+
+    def hasValidCharacters1(self, cadena):
```
please change it to lowercase and set a more appropriated name

Miquilin commented on src/com/jalasoft/search_files/utils/validation_test.py:
```diff
@@ -0,0 +1,18 @@
+import unittest
+from validations.validation import Validation
+
```
please add a new line here

Miquilin commented on src/com/jalasoft/search_files/utils/validation_test.py:
```diff
@@ -0,0 +1,18 @@
+
+class ValidationsTest(unittest.TestCase):
+    def The_value_is_a_number(self):
+        ValueExample = ["Hola", "2", "-2"]
```
please add a new line here

Miquilin replied:
please use lowercase
value_example = ...

Miquilin commented on src/com/jalasoft/search_files/utils/validation_test.py:
```diff
@@ -0,0 +1,18 @@
+        ValueExample = ["Hola", "2", "-2"]
+        message = "El valor es {Value} = {boolean}"
+        for cadena in ValueExample:
+            ValueBoolean = Validation.Isnumber(Validation, cadena)
```
please change it to lowercase
value_boolean = ...

Miquilin commented on src/com/jalasoft/search_files/utils/validation_test.py:
```diff
@@ -0,0 +1,18 @@
+from validations.validation import Validation
+
+class ValidationsTest(unittest.TestCase):
+    def The_value_is_a_number(self):
```
please change it to lowercase:
def the_value_is_a_number(self):

Miquilin commented on src/com/jalasoft/search_files/utils/validation_test.py:
```diff
@@ -0,0 +1,18 @@
+            print(message.format(Value=cadena, boolean=ValueBoolean))
+            self.assertTrue(ValueBoolean)
+
+    def The_value_is_within_a_range(self):
```
please change it to lowercase
def the_value_is_...

Miquilin commented on src/com/jalasoft/search_files/utils/validation.py:
```diff
@@ -0,0 +1,59 @@
+        pass
+
+    def Isnumber(self, cadena):
+        responseBool = cadena.isdigit()
```
please change it lowercase
response_bool = ...

Saex05 committed:
fixed name observation method and variables

mtancara committed:
done some fixes in the code

- Fixed the name of a pair of varaibles
- Fixed the unique unitest that was on validation_test.py, now it is working
